nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/security/LockSecurityPolicy.java

```diff
@@ -71,4 +76,11 @@ public boolean isExpressibleInQuery(String repositoryName) {
         return SQLQuery.Transformer.IDENTITY;
     }
 
+    /**
+     * @since 11.1
+     */
+    public static void setIgnorePolicy(boolean flagValue) {
+        ignorePolicy.set(flagValue);
```

Comment:
As an optimization, if `flagValue` is `false` then call `remove()` instead, to deallocate the value.
And actually in `isIgnorePolicy` if you get `false` you should also call `remove()` to re-clean, as the `get()` would allocate a null value in the map...